python/ray/data/dataset.py

```diff
@@ -3561,6 +3562,9 @@ def write_bigquery(
             overwrite_table: Whether the write will overwrite the table if it already
                 exists. The default behavior is to overwrite the table.
                 ``overwrite_table=False`` will append to the table if it exists.
+            enable_list_inference: Whether to enable list inference for the BigQuery.
+                Disabled by default. See details in
+                `List logical type <https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-parquet#list_logical_type>`.
```

Comment:
This link formatting will be broken - see below for fix:

loading-data-cloud-storage-parquet#list_logical_type>`_.

Reply:
Good catch! I fixed it in 06e584c415a621ba99544587dce8d0adfd06edd9